batch/job_launcher.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+import boto3
+import click
+import glob
+import os
+import re
+import tarfile
+import time
+import yaml
+
+@click.command()
+@click.option("-p", "--job-prefix", type=str, required=True,
+              help="A short but descriptive string to use as an identifier for the job run")
+@click.option("-c", "--config", "config_file", envvar="CONFIG_PATH", type=click.Path(exists=True), required=True,
+              help="configuration file for this run")
+@click.option("-j", "--num-jobs", "num_jobs", type=click.IntRange(min=1), required=True,
+              help="total number of jobs to run in this batch")
+@click.option("-s", "--sims-per-job", "sims_per_job", type=click.IntRange(min=1), required=True,
+              help="how many sims each job should run")
+@click.option("-t", "--dvc-target", "dvc_target", type=click.Path(exists=True), required=True,
+              help="name of the .dvc file that is the last step in the pipeline")
+@click.option("-i", "--s3-input-bucket", "s3_input_bucket", type=str, default="idd-input-data-sets")
+@click.option("-o", "--s3-output-bucket", "s3_output_bucket", type=str, default="idd-pipeline-results")
+@click.option("-d", "--job-definition", "batch_job_definition", type=str, default="Batch-CovidPipeline-Job")
+@click.option("-q", "--job-queue", "batch_job_queue", type=str, default="Batch-CovidPipeline")
```

Comment:
nit: for these params, i think you want `show_default=True` set so when you do a `--help` you'll see what the default value is

Reply:
also missing helpstrings :)